static/docs/commands-reference/commit.md

```diff
@@ -47,11 +49,11 @@ want. Let's take a look at what is happening in the fist scenario closely:
 Normally DVC commands like `dvc add`, `dvc repro` or `dvc run`, commit the data
 to the DVC cache as the last step. What _commit_ means is that DVC:
 
-- Computes a checksum for the file/directory.
-- Enters the checksum and file name into the DVC-file.
-- Tells the SCM to ignore the file/directory (e.g. add entry to `.gitignore`).
-  Note that if the workspace was initialized with no SCM support
-  (`dvc init --no-scm`), this does not happen.
+- Computes a checksum for the file/directory;
```

Comment:
I thought that when you start with a capital letter you have to end the sentence with `.` not `;`

Reply:
Oh umm I'm not sure but I don't think so. In general periods `.` separate full sentences whether or not they start with a capital letter (they always should anyway) and in this case the list items are just [fragments](https://college.cengage.com/english/raimes/digitalkeys/keyshtml/senten15.htm) (missing subject) anyway. `;` just helps us separate list items.

Anyway, I think its totally up to us really as tech docs aren't really covered by formal book-writing grammar )  I just applied the style we decided on recently (separate short bullet lists with `;`) and without extra line break. Note that the last sentence does have a final period `.`

Reply:
I think there are some rules - https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html :) It feels weird to see a capital-letter starting sentence end with `;` tbh.

Reply:
Those are just tips or recommendations but I don't think there are any well established rules about this. Anyway, I simply removed the semicolons as the bullet points are not full sentences in e1d6f68.

Would you like me to review once more all the bullet lists throughout the docs based on these tips? https://www.businesswritingblog.com/business_writing/2005/12/the_best_of_bul.html (and [the one you provided](https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html)) e.g. the contributing and dvcignore user guides still have full sentence bullet points separated with `;`. (I would create a new issue for this.)

Reply:
👍 yep, let's create an issue

Reply:
Done, #491